components/Header/HeaderButtonsGroup.tsx

```diff
@@ -33,13 +33,8 @@ const HeaderButtonsGroup = ({ className = '' }) => {
       <div className="pointer-events-none font-clashDisplay text-lg font-semibold text-white">
         Sonata
       </div>
-      <Image
-        src="/images/notes.jpg"
-        alt="Sonata Logo"
-        width={36}
-        height={36}
-        className={cn('rounded-full', !authenticated && 'opacity-0 pointer-events-none')}
-      />
+      {/* blank div added to keep sonata in center of header */}
```

Comment:
Remove notes from `cursorAI`
- expected: code is self-documenting
```suggestion
      {/* blank div added to keep sonata in center of header */}
```